server/backend/api/app/routers/router_smoke_test.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from typing import List
+from api.app.crud import crud_application
+from fastapi import APIRouter, Depends, Response
+from sqlalchemy.orm import Session
+from .. import database
+from api.app.models import model as models
+
+
+LOGGER = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("", status_code=200)
+def get_applications(
+    response: Response,
+    db: Session = Depends(database.get_db),
+):
+
+    """
+    List of different applications that are administered by FAM
+    """
+    LOGGER.debug(f"running router ... {db}")
+    fam_apps = db.query(models.FamApplication).all()
+    if len(fam_apps) == 0:
+        response.status_code = 417
```

Comment:
Curious on reason why using 4xx client error, not 5xx server error?

Reply:
It's not actually an error if there is no data. Just an unexpected result. Doesn't matter because anything other than 2xx will fail the smoke test.